content/W2015/W15-event-boardgames.md

```diff
@@ -19,3 +19,6 @@ Hope to see you there!
 + **What:** Board Games Meet and Greet
 + **Where:** Math CnD (MC 3002)
 + **When:** Thu. Jan. 8, 7:00&ndash;11:00PM
+
+# About Big CSters #
+Big CSters is our mentorship program for female Computer Science students. We host social activities throughout the term for CSters to meet new friends in all stages of university life.
```

Comment:
I agree that the description needs some re-wording. I just adapted the wording from the [Big CSters page](https://cs.uwaterloo.ca/wics/mentoring) on the CS website.

I think the discussion about what sort of language to use needs to be in a meeting and added to our Code of Conduct. Then, we'll take cues from there as to the language to use in ads. Once that is ratified, I'll come back to this and change the language. :)